tools/syntax/VsCode/syntaxes/bond.tmLanguage

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>fileTypes</key>
+    <array>
+        <string>bond</string>
+    </array>
+    <key>name</key>
+    <string>Bond</string>
+    <key>patterns</key>
+    <array>
+        <dict>
+            <key>include</key>
+            <string>#commonStuff</string>
+        </dict>
+        <dict>
+            <key>captures</key>
+            <dict>
+                <key>1</key>
+                <dict>
+                    <key>name</key>
+                    <string>keyword.other.namespace.source.bond</string>
+                </dict>
+                <key>2</key>
+                <dict>
+                    <key>name</key>
+                    <string>storage.type.bond</string>
+                </dict>
+                <key>3</key>
+                <dict>
+                    <key>name</key>
+                    <string>entity.name.type.namespace.source.bond</string>
+                </dict>
+            </dict>
+            <key>match</key>
+            <string>^(namespace)\s+(cpp|csharp) (.+)$</string>
```

Comment:
The `namespace foo` statement does not have to appear at the beginning of the line. Is leading/trailing whitespace already stripped?